packages/mattermost-redux/src/reducers/entities/channels.ts

```diff
@@ -708,6 +708,14 @@ function stats(state: RelationOneToOne<Channel, ChannelStats> = {}, action: Gene
     }
     case UserTypes.LOGOUT_SUCCESS:
         return {};
+    case SearchTypes.RECEIVED_SEARCH_PINNED_POSTS: {
```

Comment:
Slight nitpicking, but would you mind moving this `case` above the `LOGOUT_SUCCESS` one? We generally tend to end reducers with the logout case followed by the default one

Reply:
@hmhealey 
I fixed this as you mentioned.